src/app/App.tsx

```diff
@@ -1,12 +1,25 @@
-import { Typography } from '@mui/material';
-import { Box } from '@mui/system';
+import { Grid } from '@mui/material';
 import * as React from 'react';
+import ContentContainer from '../components/ContentContainer';
+import StartScreen from '../components/StartScreen';
+import BackgroundIMG from '../assets/backgrounds/default.jpg';
 
 function App() {
   return (
-    <Box>
-      <Typography>marsaac</Typography>
-    </Box>
+    <Grid
+      container
+      justifyContent="center"
+      alignItems="center"
+      sx={{
+        background: `url(${BackgroundIMG})`,
+        height: '100vh',
+        backgroundSize: 'cover',
+      }}
+    >
```

Comment:
Not a big deal but maybe we want this to be a `Background` component. It would just make this file a little easier to look at.

Then we could also just use that in our storybook.

If we wanted to put every story inside our background modal thing, we could make a [decorator](https://github.com/Tremors-Of-War/tremors-of-war/tree/main/src/stories/decorators) for that